### DISCUSSION TITLE
Audio Leveler / AGC

### DISCUSSION THREAD

**Discussion by tvelliott** [chuot/rdio-scanner#204] [General]
Is there a way to disable the audio leveler during conversion? The audio files I start with have a better leveler, so would like to convert them as-is. Enjoying the software! Thank you.

edit: After looking through the code a bit, I found this. I believe this is where the audio is normalized by ffmpeg. I would like to request a configuration option to disable the normalization during conversion if possible. That would make this software perfect! Thank you.

if ffmpeg.version43 {
  args = append(args, "-af", "apad=whole_dur=3s,loudnorm=I=-16:TP=-1.5:LRA=11")
}

**Comment by tvelliott** [comment]
I found that it was very easy to fork, make a change,  build the distribution without the audio normalization. Thank you!

**Comment by chuot** [comment]
Personally, I prefer Rdio Scanner's audio to stand out from the rest, but I understand that some people prefer it not. I will add an option in a future version so users can choose if they want AGC and if they want stronger AGC or not.

**Comment by tvelliott** [comment]
Thank you for the new options! Testing 6.4.1 now.
